src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBUtils.java

```diff
@@ -36,11 +36,12 @@
      * @param callsetJson path to the GenomicsDB callset JSON
      * @param vidmapJson path to the GenomicsDB vidmap JSON
      * @param vcfHeader VCF with header information to use for header lines on export
+     * @param genomicsDBOptions genotyping parameters to read from a GenomicsDB
      * @return a configuration to determine the output format when the GenomicsDB is queried
      */
     public static GenomicsDBExportConfiguration.ExportConfiguration createExportConfiguration(final File reference, final String workspace,
                                                                                                final String callsetJson, final String vidmapJson,
-                                                                                               final String vcfHeader) {
+                                                                                               final String vcfHeader, final GenomicsDBOptions genomicsDBOptions) {
```

Comment:
It's kind of weird that the reference gets passed in separately here, despite there also being a reference inside the `GenomicsDBOptions`. What if the two references were inconsistent?